.github/workflows/push.yml

```diff
@@ -77,96 +77,160 @@ jobs:
           version: v0.11.1
           registry: true
 
-      - name: Start Helm Repository (ChartMuseum)
-        run: docker run --rm -u 0 -it -d -p 8080:8080 -e DEBUG=1 -e STORAGE=local -e STORAGE_LOCAL_ROOTDIR=/charts -v $(pwd)/charts:/charts chartmuseum/chartmuseum:latest
+      - name: Start ServiceBox
```

Comment:
What is the purpose of this workflow? and difference with the `pr.yml`one?

Reply:
the PR workflow should provide a quick feedback about the acceptance of the pull request changes.
the PUSH workflow should trigger a full coverage (different java versions, run in native, etc) when changes are pushed in the main branch.

Reply:
ok. Thanks!